### DISCUSSION TITLE
Idea for Implementation to getTopSongs for Jellyfin (only one API call)

### DISCUSSION THREAD

**Discussion by joie96** [jeffvli/sonixd#426] [Ideas]
Hello,

first of all I would like to thank you for this amazing piece of software. 

I've noticed that there is no getTopSongs implementation for jellyfin. Maby you could realize this with the use of the "CommunityRating" Tag of Jellyfin items. 

I sadly don't know how to do this in TypeScript, but here is an example how the implementation looks in Python:

artistId="6f510f305c3864e9bd7e75f309e9bd70"
topSongs=client.jellyfin._get("Items", params={'userId': "{UserId}", 'IncludeItemTypes': "Audio", 'Recursive': True, 'artistIds': artistId, 'sortBy': "CommunityRating", 'sortOrder': "Descending", 'limit': 100})

This gives us the top 100 songs of the artist, based on the CommunityRating.

I would love to see this feature.  

Best regards,
Eric

**Comment by jeffvli** [comment]
Probably won't make it into Sonixd at this point, but I've just included this functionality in the [rewrite](https://github.com/jeffvli/feishin/commit/c0a703be7ae756534aff33e9a9f395c8713185f8).

**Comment by joie96** [comment]
Thank you!
